utils/validationRules/admins/createSingleAdminSchema.js

```diff
@@ -4,7 +4,7 @@ const mongoIdSchema = require("../mongoIdSchema");
 /**
  * Schema validation for POST '/admins'
  */
-const createApplicationSchema = {
+const createSingleAdminSchema = {
   headers: Joi.object({
     authorization: Joi.string().required(),
   }),
```

Comment:
You remove create admin schema validation rule. Why is that

Reply:
Original value was in error had `createApplicationSchema` instead